includes/admin/class-wcmypa-admin.php

```diff
@@ -815,18 +810,26 @@ public function getProductOptions(): array
             ],
             'Country-of-origin' => [
                 'id'          => self::META_COUNTRY_OF_ORIGIN,
-                'label'       => __('Country of origin', 'woocommerce-myparcel'),
-                'type'        => 'text',
-                'description' => wc_help_tip(__('Country of origin is required for world shipments. Defaults to shop base.', 'woocommerce-myparcel')),
+                'label'       => __('country_of_origin', 'woocommerce-myparcel'),
```

Comment:
Misschien is `product_options_country_of_origin` beter, dan is het tenminste duidelijk dat het om een productinstelling gaat. Wel lastig dat die eronder `shipment_options...` heet, maar dat is omdat dat ook daadwerkelijk een shipment option is en we die naam elders ook gebruiken